assets/blocks/Underglaze.block

```diff
@@ -4,11 +4,11 @@
     "categories" : ["Brick"],
     "template" : false,
     "shape" : "engine:cube",
-    "hardness": 50    
+    "hardness": 50,    
     "attachmentAllowed" : true,
     "supportRequired" : false,
     "penetrable" : false,
     "targetable" : true,
     "invisible" : false,
     "mass" : 40
-}
\ No newline at end of file
+}

```

Comment:
I think you commited a Windows end of line character here. Can you please double-check you editor and/or git settings.